spec/datadog/opentracer/rack_propagator_spec.rb

```diff
@@ -27,15 +27,15 @@
     before do
       # Expect carrier to be set with Datadog trace properties
       expect(carrier).to receive(:[]=)
-        .with(Datadog::Tracing::Distributed::Headers::Ext::HTTP_HEADER_TRACE_ID, trace_id.to_s)
+        .with('x-datadog-trace-id', trace_id.to_s)
```

Comment:
Instead of fixing the namespace changes, I chose to inline the string values here: it makes tests more robust and easier to read.